features/theme-list.feature

```diff
@@ -0,0 +1,64 @@
+Feature: List WordPress themes
+
+  Scenario: Refresh update_themes transient when listing themes with --force-check flag
+    Given a WP install
+    And I run `wp theme delete --all --force`
+    And I run `wp theme install --force twentytwentyfour --version=1.1`
+    And a update-transient.php file:
+      """
+      <?php
+      $transient = get_site_transient( 'update_themes' );
+      $transient->response['twentytwentyfour'] = (object) array(
```

Comment:
Using TwentyTwentyFour theme in feature test will give error in lower PHP versions. So please use may be TwentyTwelve.